woocommerce/woocommerce
# Add grouped and external products to new experiment

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Partly addresses: https://github.com/woocommerce/woocommerce/issues/42708



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to the ExPlat experiment woocommerce_product_creation_experience_add_external_and_grouped_202401_v1 and choose the variation you want to test.
2. When hovering over any Bookmarklet link of control or treatment variation, a popover should appear with instructions to try one of the variations.
3. Be sure to save the suggested link to your browser bookmarks.
4. Go to WooCommerce > Home and go through the OBW or skip.
5. Go to /wp-admin/admin.php?page=wc-admin and click on the second item to add a new product.
6. Click on the bookmark saved in step 3.
7. After clicking the saved bookmark, a modal with this title should be shown ExPlat: Successful Assignment
    Close the modal. 
8. Click on either Grouped or External product types. 
    If you chose the "control" variation, you should see the old/legacy product form.
    If you selected the "treatment" variation, you should see the blocks product editor.
9. For grouped products confirm the inventory tab is only showing the SKU field.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
